arguments.py

```diff
@@ -79,7 +79,7 @@ def parse():
     parser.add_argument('--bootargs', metavar='', type=str, default=None, help='bootargs to set or append to default args (board dependant)')
     parser.add_argument('-g', '--golden', metavar='', type=str, default=[], nargs='+', help='Path to JSON results to compare against (golden master)')
     parser.add_argument('-q', '--feature', metavar='', type=str, default=[], nargs='+', help='Features required for this test run')
-
+    parser.add_argument('-pswd', '--password', metavar='', type=str, action="append", default=None, help='password for wifi')
```

Comment:
I don't think this is the right approach. I think we should generate a random password for a sequence of tests and save it on the board object.